test/e2e/cluster_upgrade_runtimesdk.go

```diff
@@ -500,32 +473,25 @@ func runtimeHookTestHandler(ctx context.Context, c client.Client, namespace, clu
 	Eventually(func() bool {
 		return blockingCondition()
 	}, intervals...).Should(BeFalse(),
-		fmt.Sprintf("ClusterTopology reconcile did not unblock after updating hook response: hook %s still blocking", hookName))
+		fmt.Sprintf("ClusterTopology reconcile did proceed as expected when calling %s", hookName))
```

Comment:
I prefer the original text. I believe this message only shows if the check fails and the original message showed why the check failed and why.

Reply:
The issue is that the more specific message could be wrong sometimes. We don't know at this point if the hook failed to unblock or if reconciliation failed in some other way, so adding the detail makes this message kind of misleading.

Reply:
Makes sense. I would also prefer a more generic message over a misleading one

Reply:
Sounds good. I think we can keep the messaging.